src/useCases/postFeedback.ts

```diff
@@ -0,0 +1,29 @@
+import config from "lib/config"
+import getEmailer from "lib/getEmailer"
+import PromiseResult from "types/PromiseResult"
+import logger from "utils/logger"
+
+const postFeedback = (feedback: string, currentUserEmail: string): PromiseResult<void> => {
+  const sendFeedbackTo = "kayleigh.derricutt@madetech.cjsm.net" // sorry Kayleigh :(
```

Comment:
Does Kayleigh have a CJSM account? I'm not sure she does...

Reply:
she does. I have send her some emails in the past :D